Lib/test/test_asyncio/test_tasks.py

```diff
@@ -3288,13 +3288,12 @@ async def coro(s):
             self.one_loop.call_exception_handler.assert_not_called()
 
 
-class RunCoroutineThreadsafeTests(test_utils.TestCase):
+class RunCoroutineThreadsafeTests(unittest.IsolatedAsyncioTestCase):
```

Comment:
These changes need to be reverted - they are unrelated to the current issue, and they also caused a test failure.